components/Status.tsx

```diff
@@ -81,3 +81,23 @@ export default function Status({ statusData, coordinatorName }) {
     </li>
   );
 }
+
+
+function scriptTypeToString(scriptType: string): string {
```

Comment:
this function can go in `utils/convertor.ts` and then we can also use it in the `InputType` component

Reply:
Done